app/server/models/transfer_account.py

```diff
@@ -138,6 +139,11 @@ def primary_user(self):
     def primary_user_id(self):
         return self.primary_user.id
 
+    # rounded balance
+    @hybrid_property
+    def account_balance(self):
+        return (self._balance_wei or 0) / int(1e18)
```

Comment:
yes! i'll have to add some argument formatting on the client to compensate because i'm thinking users will only enter whole values so we'll have to convert them into cents